wwwroot/src/App.js

```diff
@@ -43,24 +49,38 @@ const theme = createMuiTheme({
   palette: {
     primary: { 
       main: '#FF6E4E',
-      textPrimary: '#8B8B8B',
+      textPrimary: '#5F5F5F',
```

Comment:
Hang on, as per the wireframes, the color should be #8B8B8B, shouldn't it be?

Reply:
Which wireframes are you referring to? I've been looking at Aezha's Figma project and I see #5F5F5F everywhere

<img width="1438" alt="Screen Shot 2020-04-13 at 3 51 37 PM" src="https://user-images.githubusercontent.com/53886065/79155356-fbbd1880-7d9e-11ea-8437-348c4ab0e3d5.png">
<img width="1438" alt="Screen Shot 2020-04-13 at 3 52 04 PM" src="https://user-images.githubusercontent.com/53886065/79155363-fe1f7280-7d9e-11ea-901a-ad45ebe1f908.png">

Reply:
on Signup and Login pages it's a different color. We need Aezha to make a decision on which color she wants us to use.

Reply:
I've changed the color back to #8B8B8B for now - we can easily change it again when we hear back from Aezha on this.